app/poros/movie.rb

```diff
@@ -0,0 +1,13 @@
+class Movie
```

Comment:
Absolutely amazing! Very clean and readable! Great job on this refactor!

Reply:
Yeah, but it's missing run time :-(